configs/stac/stac_fly_tethered.yaml

```
@@ -0,0 +1,18 @@
+fit_path: "Fruitfly_fit.p"
+transform_path: "transform_mocap_fly.h5"
+# This data is unpublished, need to speak to Elliot Abe about how to obtain it.
+# A possible source is https://datadryad.org/stash/dataset/doi:10.5061/dryad.nzs7h44s4
+# See the discussion here for more info: 
```

yamllint

[error] trailing spaces

(trailing-spaces)